modules/mbedtls/Kconfig

```
@@ -145,6 +145,13 @@ config MBEDTLS_DEBUG_EXTRACT_BASENAME_DISABLED
 
 endchoice
 
+config MBEDTLS_DEBUG_STRIP_NEWLINE
+	bool "Strip newline from logged string"
```

minor proposal, consider just this in title
```suggestion
 	bool "Strip newlines" 
```
we are already in a sub-menu related to mbed TLS debugging, so from where else would you be stripping newlines ?
Details are in the help text.